Specify project dependencies (#8418)
fishtown-analytics/docs.getdbt.com: website/docs/docs/mesh/govern/project-dependencies.md

<UseCaseInfo/>

> ## How do I specify project dependencies?

Reviewer: Unless we're doing some sort of FAQ section, I recommend phrasing the headers as a statement rather than a question:
```suggestion
## Define project dependencies
```

> If your dbt project relies on models from another project, you can define that relationship using project dependencies. The following steps will walk you through how to specify project dependencies in dbt.

Reviewer: ```suggestion
If your dbt project relies on models from another project, you can define that relationship using project dependencies. The following steps walk you through specifying project dependencies in dbt:
```

1. Create a file called `dependencies.yml` at the root of your dbt project.

> 2. In the `dependencies.yml`, list the upstream dbt project your project depends on.

Reviewer: Could be helpful to sprinkly in some of the finer details that we go into more in the example
```suggestion
2. In the `dependencies.yml`, list the upstream dbt project your project depends on as they appear in the `dbt_projects.yml` file.
```

3. (Optional) Define the specific models you expect from that upstream project to make the dependency explicit.
4. Use [`ref()`](/reference/dbt-jinja-functions/ref) with the project name to reference upstream models in your SQL.
5. Commit the changes and ensure the dependency is configured in <Constant name="cloud" />.

> 6. dbt will resolve the dependency, build upstream projects first, and show cross-project lineage.

Reviewer: Can we specify where they can view the cross-project lineage?

> ## Example

Reviewer: Suggest making this an H3 header as a subsection of the newly created section above since these two parts really go hand-in-hand

> 6. dbt will resolve the dependency, build upstream projects first, and show cross-project lineage.

Author: ```suggestion
6. dbt will resolve the dependency, ensure upstream projects are built first, and surface cross-project lineage in the lineage and DAG (Directed Acyclic Graph) views.
```

> ## Example

Author: ```suggestion
### Example
```
